core/src/main/java/hudson/model/Descriptor.java

```diff
@@ -1198,7 +1198,7 @@ public String getFormField() {
         @Override
         public void generateResponse(StaplerRequest req, StaplerResponse rsp, Object node) throws IOException, ServletException {
             if (FormApply.isApply(req)) {
-                FormApply.applyResponse("notificationBar.show(" + quote(getMessage()) + ",notificationBar.ERROR)")
+                FormApply.showNotification(quote(getMessage()), FormApply.NotificationType.ERROR)
```

Comment:
This looks wrong. The `#quote` call seems to have been needed because this built a JavaScript string and needed to escape e.g. `"`. That's no longer the case, so the call should be removed.